doc/usage.md

```diff
@@ -206,18 +218,24 @@ nil
 #'user/server
 ```
 
+If you want your nREPL server to listen on a particular address instead of the
+default one, you can use the `:bind` keyword to specify the address to
+listen on. E.g., to make the nREPL server listen on address 172.18.0.5
+and port 4001: 
+
+```clojure
+=> (use '[nrepl.server :only (start-server stop-server)])
+nil
+=> (defonce server (start-server :bind "172.18.0.5" :port 4001))
+#'user/server
+```
+
 Depending on what the lifecycle of your application is, whether you want to be
 able to easily restart the server, etc., you might want to put the value
 `start-server` returns into an atom or somesuch.  Anyway, once your app is
 running an nREPL server, you can connect to it from a tool like Leiningen or
-Counterclockwise or Reply, or from another Clojure process:
-
-```clojure
-=> (with-open [conn (repl/connect :port 7888)]
-     (-> (repl/client conn 1000)
-       (repl/message {:op :eval :code "(+ 1 1)"})
-       repl/response-values))
-```
+Counterclockwise or Reply, or from another Clojure process, as shown
+in [Connecting to an nREPL Server](#connecting-to-an-nrepl-server).
```

Comment:
Instead of showing how to connect to a running nREPL server again (and add examples for local and remote servers again) I think it's better to link to the section when we already explain all this.